CMakeLists.txt

```diff
@@ -340,6 +340,36 @@ install(FILES
     ${CMAKE_CURRENT_BINARY_DIR}/${PACKAGE_NAME}-config-version.cmake
   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PACKAGE_NAME})
 
+# Create pkg-config file
+include(build/JoinPaths.cmake)
+# from: https://github.com/jtojnar/cmake-snips#concatenating-paths-when-building-pkg-config-files
+join_paths(DIRECTXTEX_INCLUDEDIR_FOR_PKG_CONFIG "\${prefix}" "${CMAKE_INSTALL_INCLUDEDIR}")
+join_paths(DIRECTXTEX_LIBDIR_FOR_PKG_CONFIG "\${prefix}"     "${CMAKE_INSTALL_LIBDIR}")
+
+set(DIRECTXTEX_DEP_L "")
+if(ENABLE_OPENEXR_SUPPORT)
+  list(APPEND DIRECTXTEX_DEP_L "OpenEXR")
```

Comment:
@JPeterMugaas The OpenEXR used by vcpkg does NOT install a .pc file, so this fails right now...

Reply:
The OpenEXR CMakeLists.txt file has a "OPENEXR_INSTALL_PKG_CONFIG" option that should be to ON.

If you have to make your own .pc file, there's a guide at: https://people.freedesktop.org/~dbn/pkg-config-guide.html as well as the man page, https://manpage.me/?pc .   If it helps, here's the contents of an OpenEXR.pc on my computer's mingw-w64 setup:
```
## SPDX-License-Identifier: BSD-3-Clause
## Copyright (c) Contributors to the OpenEXR Project.
##

prefix=/mingw64
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include
OpenEXR_includedir=${includedir}/OpenEXR
libsuffix=-3_2

Name: OpenEXR
Description: OpenEXR image library
Version: 3.2.1

Libs:  -L${libdir} -lOpenEXR -lOpenEXRUtil -lOpenEXRCore -lIex -lIlmThread
Cflags: -I${includedir} -I${OpenEXR_includedir} -pthread
Requires: Imath

```
and from my ArchLinux setup:
```
##
## SPDX-License-Identifier: BSD-3-Clause
## Copyright (c) Contributors to the OpenEXR Project.
##

prefix=/usr
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include
OpenEXR_includedir=${includedir}/OpenEXR
libsuffix=-3_2

Name: OpenEXR
Description: OpenEXR image library
Version: 3.2.1

Libs:  -L${libdir} -lOpenEXR${libsuffix} -lOpenEXRUtil${libsuffix} -lOpenEXRCore${libsuffix} -lIex${libsuffix} -lIlmThread${libsuffix}
Cflags: -I${includedir} -I${OpenEXR_includedir} -pthread
Requires: Imath

```
Those are my thoughts for the moment.

Reply:
Thanks. Using ``-DOPENEXR_INSTALL_PKG_CONFIG=ON`` in the port fixed it. Much appreciated.

> Note that vcpkg still has OpenEXR 3.1.